### DISCUSSION TITLE
Normal italic letter cropped off with cell_width 95%

### DISCUSSION THREAD

**Discussion by barraIhsan** [kovidgoyal/kitty#7931] [Q&A]
```
# Fonts
font_family JetBrainsMono Nerd Font
font_size 18.0
modify_font cell_width 95%
```

Running neovim:
![image](https://github.com/user-attachments/assets/41f8b4e6-639c-49d7-9c41-b12b7ea4784b)

When I set it to `100%`, it still *slightly* cropped off
![image](https://github.com/user-attachments/assets/01f9d3c3-5be1-4d0e-830d-7822ffb95c6e)

The reason why I set it to be 95% at the first place because I feel like kitty's default font is too spaced out?

**Comment by kovidgoyal** [comment]
Not sure what you are asking here. If you make the cell too narrow for
letters to fit they will be either cropped or resized and mis-aligned.

**Reply by barraIhsan** [reply]
yes, but even with the `cell_width` being set to `100%`, it still getting cropped. What's the default value?

**Comment by kovidgoyal** [comment]
If it gets cropped at 100% then the font you are using does not have an
italic face whose metrics are compatible with its normal face, so you
will need to either use a proper monospaced font or widen the cells.

**Reply by barraIhsan** [reply]
![image](https://github.com/user-attachments/assets/22db078d-afb7-4c9d-b0b1-b87722d3e0cd)
oh, the right side is cropped?
https://www.jetbrains.com/lp/mono/#font-family

But strangely, in other terminal emulator (and code editor like vscode), it works fine with default setting (probably 100% cell_width)
